rooms.js

```diff
@@ -906,6 +941,8 @@ let BattleRoom = (() => {
 	BattleRoom.prototype.expireTimer = null;
 	BattleRoom.prototype.active = false;
 
+	BattleRoom.prototype.modlog = fs.createWriteStream(path.resolve(__dirname, 'logs/modlog/modlog_battle.txt'), {flags:'a+'});
```

Comment:
Well, this can't be rewritten to ``class`` notation. Not that I care, but probably @Zarel does.